.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules/
```

Comment:
Mind keeping this out of the PR? I don't use local `.gitignore` in projects - language/platform-related ignores I prefer in global config.